lib/Indexer/Adapter/Tolerant/Indexer/ClassDeclarationIndexer.php

```diff
@@ -54,7 +54,7 @@ public function indexAttributes(ClassRecord $record, ClassDeclaration $node): vo
                     continue;
                 }
                 /** @phpstan-ignore-next-line */
-                if ($attribute->name?->getText() === 'Attribute') {
+                if ((string) $attribute->name?->getResolvedName() === \Attribute::class) {
```

Comment:
[getResolvedName](https://github.com/microsoft/tolerant-php-parser/blob/main/src/Node/QualifiedName.php#L82) is however potentially enormously inefficient. For this case it's probably enough to just check both variations?

Reply:
Btw. I did not checked how much the indexing time increases after this change.

Reply:
would be relative to the numnber of attributes in the source tree i guess

Reply:
True, it could be expensive. I haven't thought about it too much and just having a "good enough" solution would probably be sufficient.

Reply:
Independently of expensiveness I added such example to the test.